kit/Kit.cpp

```diff
@@ -1799,6 +1803,21 @@ class Document final : public DocumentManagerInterface
     bool _inputProcessingEnabled;
 };
 
+#if !defined FUZZER && !defined BUILDING_TESTS
+
+// When building the fuzzer we link LOOLWSD.cpp into the same executable so the
+// Protected::addOneRecording() there gets used. When building the unit tests the one in
+// ProfileZone.cpp gets used.
+
+void ProfileZone::addOneRecording(const std::string &sRecording)
+{
+    if (!config::getBool("trace_event[@enable]", false))
+        return;
+
+    singletonDocument->sendTextFrame("trace: \n" + sRecording);
```

Comment:
Hmm - we really want to send these one by one? or does the recording contain lots of back to back events? We really want to reduce the performance impact and overhead of performance tracing as much as possible so it doesn't distort what we're trying to measure.

Reply:
Yeah, clearly we need to optimise this before even seeing whether it has any impact. Will get right on it.